### DISCUSSION TITLE
Update of minor Angular version provides wrong zone.js version

### DISCUSSION THREAD

**Discussion by BeCaDRI** [renovatebot/renovate#25282] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

Hey,

we have the following config:

`{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "schedule:earlyMondays"
  ],
  "ignorePresets": [
    "schedule:weekdays"
  ],
  "commitBody": "[skip ci]",
  "includePaths": [
    "app/myapp/**",
    "app/catalog/**",
    "app/app-mat-library-workspace/**"
  ],
  "rebaseWhen": "conflicted",
  "packageRules": [
    {
      "extends": "monorepo:angular",
      "groupName": "angular monorepo",
      "addLabels": [
        "angular"
      ],
      "prPriority": 2,
      "assignees": [
        "test"
      ]
    }
  ]
}`

We use this to update a Angular Repo where are three Angular Apps are included (everyone runs on the same Angular Version). It works fine so far, but what is strange, that renovate proposes an update from Angular 15.2.9 to 15.2.10 => fine.
But at the same time it proposes to update zone.js to V0.14.
Problem is that Angular 15 is not compatible to Zone.js V0.14. 

`"peerDependencies": {
    "rxjs": "^6.5.3 || ^7.4.0",
    "zone.js": "~0.11.4 || ~0.12.0 || ~0.13.0"
  },`

Should renovate not be able to detect this peerdependency?

Thanks for any help.
Regards

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by rarkins** [comment]
Renovate does not have the capability to resolve peerDependencies to check compatibility like that. Are these from the same monorepo but incompatible with each other?

**Reply by BeCaDRI** [reply]
Okay, when it can't detect peerDependencies I need to group zone.js or use a fixed max version in the config I suppose? Otherwise it will always update some packages out of range of the peerdependencies of Angular? Correct?

What does you question relate do? I don't get the context..

**Reply by rarkins** [reply]
If you wish to exclude zone.js from the angular monorepo grouping then you can give it no group like this:

```
"packageRules": [{
  "matchPackageNames": ["zone.js"],
  "groupName": null
}]
```

**Reply by BeCaDRI** [reply]
It should be in the or at least I should give it a max version as far as I understand. My previous assumption was, that renovate is able to detect peerDependencies inside a monorepo. As it is not, I need to update my config to use the max versions. 
Not perfect, but will still do it. Thanks
